ha/resource/resource_agent.py

```diff
@@ -298,21 +292,15 @@ def stop(self):
         Log.debug(f"In stop for {filename}")
         if os.path.exists(const.HA_INIT_DIR + filename):
             os.remove(const.HA_INIT_DIR + filename)
-        while True:
-            if self.monitor() == const.OCF_NOT_RUNNING:
-                time.sleep(2)
-                break
-        if self.monitor(state=const.STATE_STOP) == Action.RESTART:
-            Log.debug(f"Restarting {filename} resource")
-            time.sleep(2)
-            return const.OCF_SUCCESS
+            Log.debug(f"Stopping {filename} resource")
+        Log.debug(f"Stopped {filename} resource return success")
         return const.OCF_SUCCESS
 
     def metadata(self):
         """
         Provide meta data for resource agent and parameter
         """
-        env=r'''
+        env=u"""
```

Comment:
what are these changes ?

Reply:
i tried this to resolve meta-data issue but as it still exist reverted it

https://jts.seagate.com/browse/EOS-13879